### ISSUE TITLE
Windows is missing console output

### ISSUE THREAD

**Issue by greenfork** [greenfork/nimraylib_now#32]

**Comment by stefandd** [comment]
In raylib_build_static.nim you pass the following option:
```Nim
when defined(windows) and not defined(emscripten):
  {.passL: "-lgdi32".}
  {.passL: "-lopengl32".}
  {.passL: "-lwinmm".}
  {.passL: "-Wl,--subsystem,windows".}
```
Compiling for the windows subsytem completely kills all echo and engine output. It is great for production maybe, but not during development
```Nim
when defined(windows) and not defined(emscripten):
  {.passL: "-lgdi32".}
  {.passL: "-lopengl32".}
  {.passL: "-lwinmm".}
 #{.passL: "-Wl,--subsystem,windows".}```
 Commenting it out, brings back all the output. The other problem with compiling to the windows subsystem is, as I had already said, that all the Nim runtime error messages and assertion messages are gone too. I recommend to not default to this linker option
Please go ahead and close the issue, or if you concur, fix it as suggested. Thanks for you quick reply anyway!

**Comment by greenfork** [comment]
@stefandd thanks a lot for the fix!

Fixed in https://github.com/greenfork/nimraylib_now/commit/f1d03904153303da9526e8be2f113c3001f1cc71 and released a new version v0.13.1
